src/rt/cover.d

```diff
@@ -57,9 +58,35 @@ private
     __gshared
     {
         Cover[] gdata;
+        Config config;
+    }
+
+    struct Config
+    {
         string  srcpath;
         string  dstpath;
         bool    merge;
+
+    @nogc nothrow:
+
+        bool initialize()
+        {
+            return initConfigOptions(this, this.errorName);
+        }
+
+        void help()
+        {
+            string s = "Code coverage options are specified as white space separated assignments:
+    merge:0|1      - 0 overwrites existing reports, 1 merges current run with existing coverage reports (default: %d)
+    dstpath:<PATH> - write code coverage reports to <PATH> (default: current
```

Comment:
You use the third person for the explanation of the merge option - consistency would be great.